src/components/avatar.vue

```diff
@@ -86,16 +86,16 @@ export default defineComponent({
 });
 </script>
 <style scoped>
-.profile-wrap {
+.profile-wrap { width: 100% !important; height: auto !important;
```

Comment:
Let's please stick with one item per line as the other css is formatted (here and in the other changes, too)